js_modules/dagit/packages/core/src/runs/RunTable.tsx

```diff
@@ -148,6 +148,7 @@ export const RUN_TABLE_RUN_FRAGMENT = gql`
     canTerminate
     mode
     rootRunId
+    runConfigYaml
```

Comment:
I think this can be expensive, depending on the size of the config document.  Is there a way to defer fetching until there's some intent to re-execute?

Reply:
Ahh yeah if IRC we recently had a customer say their run config was like 100kb of YAML - I know our pagination limits the impact to just 25 runs, but since this page auto-refreshes it might add up over time. Probably worth deferring 😬